errbot/templates/initdir/config.py.tmpl

```diff
@@ -0,0 +1,18 @@
+import logging
+
+# This is a minimal configuration to get you started with the Text mode.
+# If you want to connect Errbot to chat services, checkout
+# the options in the more complete config-template.py from here:
+# https://raw.githubusercontent.com/errbotio/errbot/master/errbot/config-template.py
+
+BACKEND = 'Text'  # Errbot will start in text mode (console only mode) and will answer commands from there.
+
+BOT_DATA_DIR = '{{ data_dir }}'
+BOT_EXTRA_PLUGIN_DIR = '{{ extra_plugin_dir }}'
+BOT_LOG_FILE = '{{ log_path }}'
+BOT_LOG_LEVEL = logging.DEBUG
+
+BOT_ADMINS = ('CHANGE ME', )
+AUTOINSTALL_DEPS = True
+
+TEXT_DEMO_MODE = False  # Put that to True if you want to suppress the debug output in Text mode.
```

Comment:
I think we should default to true here. I think that's a much nicer user experience by default as the bot output is a lot easier to read like that.